test/PowerShellEditorServices.Test/Debugging/DebugServiceTests.cs

```diff
@@ -1,1005 +1,1046 @@
-        {
-            Task executeTask =
-                this.powerShellContext.ExecuteScriptStringAsync(
-                    this.debugScriptFile.FilePath);
-
-            // Break execution and wait for the debugger to stop
-            this.debugService.Break();
-            await this.AssertStateChange(
-                PowerShellContextState.Ready,
-                PowerShellExecutionResult.Stopped);
-
-            // Try running a command from outside the pipeline thread
-            await this.powerShellContext.ExecuteScriptStringAsync("Get-Command Get-Process");
-
-            // Abort execution and wait for the debugger to exit
-            this.debugService.Abort();
-
-            await this.AssertStateChange(
-                PowerShellContextState.Ready,
-                PowerShellExecutionResult.Stopped);
-        }
-
-        [Fact]
-        public async Task DebuggerVariableStringDisplaysCorrectly()
-        {
-            await this.debugService.SetLineBreakpointsAsync(
-                this.variableScriptFile,
-                new[] { BreakpointDetails.Create("", 18) });
-
-            // Execute the script and wait for the breakpoint to be hit
-            Task executeTask =
-                this.powerShellContext.ExecuteScriptStringAsync(
-                    this.variableScriptFile.FilePath);
-
-            await this.AssertDebuggerStopped(this.variableScriptFile.FilePath);
-
-            StackFrameDetails[] stackFrames = debugService.GetStackFrames();
-
-            VariableDetailsBase[] variables =
-                debugService.GetVariables(stackFrames[0].LocalVariables.Id);
-
-            var var = variables.FirstOrDefault(v => v.Name == "$strVar");
-            Assert.NotNull(var);
-            Assert.Equal("\"Hello\"", var.ValueString);
-            Assert.False(var.IsExpandable);
-
-            // Abort execution of the script
-            this.powerShellContext.AbortExecution();
-        }
-
-        [Fact]
-        public async Task DebuggerGetsVariables()
-        {
-            await this.debugService.SetLineBreakpointsAsync(
-                this.variableScriptFile,
-                new[] { BreakpointDetails.Create("", 14) });
-
-            // Execute the script and wait for the breakpoint to be hit
-            Task executeTask =
-                this.powerShellContext.ExecuteScriptStringAsync(
-                    this.variableScriptFile.FilePath);
-
-            await this.AssertDebuggerStopped(this.variableScriptFile.FilePath);
-
-            StackFrameDetails[] stackFrames = debugService.GetStackFrames();
-
-            VariableDetailsBase[] variables =
-                debugService.GetVariables(stackFrames[0].LocalVariables.Id);
+// //
+// // Copyright (c) Microsoft. All rights reserved.
+// // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// //
+
+// using Microsoft.PowerShell.EditorServices.Utility;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4993607)